core/controllers/subtopic_viewer.py

```diff
@@ -26,6 +27,25 @@
 class SubtopicViewerPage(base.BaseHandler):
     """Renders the subtopic viewer page."""
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'classroom_url_fragment': constants.SCHEMA_FOR_CLASSROOM_URL_FRAGMENTS,
+        'topic_url_fragment': constants.SCHEMA_FOR_TOPIC_URL_FRAGMENTS,
+        'subtopic_url_fragment': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_regex_matched',
+                    'regex_pattern': '[a-z]+(-[a-z]+)*$'
```

Comment:
I think we have a constant for this value in the codebase. Can you please try `constants.VALID_URL_FRAGMENT_REGEX`

Reply:
@sahiljoster32 
Done! PTAL!